pcm-player.js

```diff
@@ -85,6 +85,25 @@ PCMPlayer.prototype.createContext = function() {
     });
 };
 
+PCMPlayer.prototype.createContextWithElement = function() {
+    this.audioCtx = new (window.AudioContext || window.webkitAudioConext)();
+    this.webAudioTouchUnlock(this.audioCtx).then(function () {
+        const destination = this.audioCtx.createMediaStreamDestination();
+        this.gainNode = this.audioCtx.createGain();
+        this.gainNode.gain.value = this.options.gain;
+        this.gainNode.connect(destination);
```

Comment:
My main feedback would be to extract and reuse the copy-pasted code from createContext, so we only have one place to make changes

Reply:
Yup, good call.